mobile_back_qti/cpp/backend_qti/settings/qti_settings_default_dsp.pbtxt

```diff
@@ -0,0 +1,159 @@
+# proto-file: flutter/cpp/proto/backend_setting.proto
+# proto-message: BackendSetting
+
+common_setting {
+  id: "num_threads"
+  name: "Number of threads"
+  value {
+    value: "4"
+    name: "4 threads"
+  }
+}
+
+common_setting {
+  id: "profiling_level"
+  name: "profile level"
+  value {
+    value: "off"
+    name: "profile level"
+  }
+}
+
+benchmark_setting {
+  benchmark_id: "image_classification"
+  accelerator: "snpe_dsp"
+  accelerator_desc: "HTP"
+  framework: "SNPE"
+  custom_setting {
+    id: "bg_load"
+    value: "true"
+  }
+  custom_setting {
+    id: "use_ion_buffer"
+    value: "false"
+  }
+  model_path: "https://github.com/mlcommons/mobile_models/releases/download/v3.0-snpe/mobilenet_edgetpu_224_1.0_htp.dlc"
+  model_checksum: "2e7c4d33b480b5566bdf05e1204b6152"
```

Comment:
Can you please migrate all the old `benchmark_setting` to the new schema with `delegate_choice`?